### PR TITLE
WMCO 9.0.0 release notes

### PR DESCRIPTION
**Pull request by DCChadwick**
Windows Machine Config Operator 9.0.0 Release Notes

Version(s):

4.14

Issue:

[OSDOCS-6623](https://issues.redhat.com//browse/OSDOCS-6623)

Link to docs preview:

https://68795--docspreview.netlify.app/openshift-enterprise/latest/windows_containers/windows-containers-release-notes-9-x#wmco-prerequisites_windows-containers-release-notes

QE review:
- [x] QE has approved this change.

Additional information:

This PR replaces: https://github.com/openshift/openshift-docs/pull/61644

### CODE REVIEW

File: windows_containers/windows-containers-release-notes-9-x.adoc [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+These release notes track the development of the WMCO, which provides all Windows container workload capabilities in {product-title}.
+
+ifndef::openshift-origin[]
+[id="getting-support"]
+== Getting support
+
+// wording taken and modified from https://access.redhat.com/support/policy/updates/openshift#windows
+
+Windows Container Support for Red Hat OpenShift is provided and available as an optional, installable component. Windows Container Support for Red Hat OpenShift is not part of the {product-title} subscription. It requires an additional Red Hat subscription and is supported according to the link:https://access.redhat.com/support/offerings/production/soc/[Scope of coverage] and link:https://access.redhat.com/support/offerings/production/sla[Service level agreements].
```
[the remark this answers is not in the document]
Comment (abrennan89) [reply windows_containers/windows-containers-release-notes-9-x.adoc:22]: If the product name is "Windows Container Support for Red Hat OpenShift", I'd strongly recommend adding an attribute for this to make it easier for future reviewers 🙂 

At first I didn't realize this is probably the product name, so I added suggestions to use the `{product-title}` attribute, but I realize now that might not be correct for this case.

**Review by abrennan89 — COMMENTED** [review]
Some questions and suggestions

**Review by jeana-redhat — COMMENTED** [review]
A couple rendering issues remain, as well as a bit of peer review feedback that needs to be addressed. Very nearly there :)

/remove-label merge-review-in-progress
/remove-label merge-review-needed

**Comment by DCChadwick** [comment]
Addressed merge review concerns. Re-applying the merge-review-needed label.

**Review by adellape — APPROVED** [review]
Leaving some comments for completeness, but not blocking merge. Also possibly consider the [earlier comment](https://github.com/openshift/openshift-docs/pull/68795/files#r1414336463) about breaking out a `wmco-9.0.0.adoc`-style module in the future.

**Pull request merged**